src/components/UI/Inputs/CurrencyInput/index.tsx

```diff
@@ -149,6 +174,14 @@ CurrencyInput.propTypes = {
    * The value change event handler.
    */
   onChange: PropTypes.func,
+  /**
+   * An error message to be displayed near the input.
+   */
+  validationError: PropTypes.string,
+  /**
+   * Whether the input can be modified or not.
+   */
+  disabled: PropTypes.bool,
```

Comment:
Added support for displaying validation errors and for displaying if the input is disabled or not